buzsaki_lab_to_nwb/tingley_code_septal/tingleyseptal_utils.py

```diff
@@ -0,0 +1,17 @@
+"""Authors: Heberto Mayorquin and Cody Baker."""
+from mat73 import loadmat as loadmat_mat73
+from mat4py import loadmat as loadmat_mat4py
+from scipy.io import loadmat as loadmat_scipy
+
+
+def read_matlab_file(file_path):
+    file_path = str(file_path)
+
+    try:
+        mat_file = loadmat_mat4py(str(file_path))
+    except:
+        try:
+            mat_file = loadmat_mat73(file_path)
+        except:
+            mat_file = loadmat_scipy(file_path)
+    return mat_file
```

Comment:
Just as a heads up, while this kind of function will 'always work', each of those packages seems to have a different API structure for accessing internal values from the files, so any time you debug your behavior interfaces on a single file, you're debugging it for that API. But as we've sometimes seen, the version might change over a single session and so if you're not checking for that you might see missing keys simply because the file got loaded by a different reader.

Reply:
Yes. I think I will allocate some time to think better on how to consistently access matlab files in the same way. Right now I ran this for all the session and it seems to be working but I agree with you is a rather brittle solution.